modules/calib3d/test/test_solvepnp_ransac.cpp

```diff
@@ -61,22 +61,22 @@ class CV_solvePnPRansac_Test : public cvtest::BaseTest
         eps[SOLVEPNP_DLS] = 1.0e-2;
         eps[SOLVEPNP_UPNP] = 1.0e-2;
         totalTestsCount = 10;
+        pointsCount = 500;
     }
     ~CV_solvePnPRansac_Test() {}
 protected:
-    void generate3DPointCloud(vector<Point3f>& points, Point3f pmin = Point3f(-1,
-        -1, 5), Point3f pmax = Point3f(1, 1, 10))
+    void generate3DPointCloud(vector<Point3f>& points,
+        Point3f pmin = Point3f(-10, -10, 5),
+        Point3f pmax = Point3f(10, 10, 10))
     {
-        const Point3f delta = pmax - pmin;
+        RNG rng(0x99999999); // fix the seed to use "fixed" input 3D points
```

Comment:
I believe we should not fix point cloud between different test cases. Try to use `cv::theRNG()` or `ts->get_rng()` instead.